doorstop/core/publisher.py

```diff
@@ -386,7 +385,7 @@ def _lines_markdown(obj, **kwargs):
                 yield ""  # break before links
                 items2 = item.parent_items
                 if settings.PUBLISH_CHILD_LINKS:
-                    label = "Parent links:"
+                    label = "🡩 Parents:"
```

Comment:
@jacebrowning Do you like these arrows?

Reply:
Is https://github.com/doorstop-dev/doorstop/pull/353#discussion_r300816685 still an issue for you?

Reply:
I don't see an arrow:

![](https://zappy.zapier.com/0cd51d456e70ebca49dbd3ede884cba5.png)

Reply:
Strange, feels like a GitHub or browser rendering bug. It renders fine for me using Chrome and Windows 10, but I see the same as @jacebrowning using Chrome on Android. Here's what I see: 
![image](https://user-images.githubusercontent.com/11055068/76809955-7b64c100-67ba-11ea-9c02-3458f2a73a08.png)

Reply:
Maybe we'll just remove the arrows for now. I put them on as visual candy anyway.

Reply:
It's a question of whether or not there are issues rendering the published documents. I don't think it should be a problem, but as you say, it's nonfunctional. I'll leave it up to @jacebrowning.